Sources/SwiftInspectorVisitors/ProtocolVisitor.swift

```diff
@@ -56,16 +56,18 @@ public final class ProtocolVisitor: SyntaxVisitor {
     let typealiasVisitor = TypealiasVisitor(parentType: .simple(name: name))
     typealiasVisitor.walk(node.members)
 
+    let propertiesVisitor = PropertySyntaxVisitor()
+    propertiesVisitor.walk(node.members)
+
     protocolInfo = ProtocolInfo(
       name: name,
       associatedTypes: associatedtypeVisitor.associatedTypes,
       inheritsFromTypes: typeInheritanceVisitor.inheritsFromTypes,
       genericRequirements: genericRequirementVisitor.genericRequirements,
       modifiers: .init(declarationModifierVisitor.modifiers),
-      innerTypealiases: typealiasVisitor.typealiases)
+      innerTypealiases: typealiasVisitor.typealiases,
+      properties: propertiesVisitor.propertiesInfo)
```

Comment:
non-blocking: thoughts on renaming the `propertiesInfo` getter to `properties` for consistency?

Reply:
https://github.com/fdiaz/SwiftInspector/pull/100/commits/56c30fdd452c17a11856f905e974e4c253b13eae